### ISSUE TITLE
Data for MICCAI

### ISSUE THREAD

**Issue by fepegar** [DeepRegNet/Benchmark#9]
The data is in `/home/fernando/datasets/deepreg`.
You will find there four directories:

1. `matrices`: affine matrices (.txt, 12 DOF) from native space to MNI
2. `mri`: images already in MNI space
3. `gif_parcellation`: GIF parcellations in native space
4. `reference`: MNI-ICBM-152 template and corresponding GIF parcellation. The MRI needs to be skull-stripped using the parcellation.

You can visualise parcellations on 3D Slicer or NifTK using [this colour table](https://github.com/thenineteen/Semiology-Visualisation-Tool/blob/master/slicer/Resources/Color/BrainAnatomyLabelsV3_0.txt).

I suggest using [TorchIO](https://torchio.readthedocs.io/) for all that needs to be done. Here's some code that might help:

```python
import torchio as tio

matrix_path, parcellation_path, mri_mni_path = _ # of a subject
mri_mni_skull_stripped_path = _
parcellation_mni_path = _

# Resample parcellation to MNI
matrix = tio.io.read_matrix(matrix_path)
parcellation = tio.LabelMap(parcellation_path, to_mni=matrix)
resample = tio.Resample(mri_mni_path, pre_affine_name='to_mni')
parcellation_mni = resample(parcellation)
parcellation_mni.save(parcellation_mni_path)

# Get brain mask
SMALLEST_BRAIN_LABEL = 24  # from colour table
extract_brain = tio.Lambda(lambda x: (x >= SMALLEST_BRAIN_LABEL))
brain_mask = extract_brain(parcellation_mni)

# Skull-stripping
def skull_strip(image, mask):
    image.data[~mask.data.bool()] = 0

mri = tio.ScalarImage(mri_mni_path)
skull_strip(mri, brain_mask)
mri.save(mri_mni_skull_stripped_path)
```

You can use similar code to perform skull-stripping on the template.

If you want to use the same brain structures for evaluation, you'll need to find a mapping between [FreeSurfer labels](https://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/AnatomicalROI/FreeSurferColorLUT) and the corresponding labels from the colour table I shared above.

**Comment by mathpluscode** [comment]
~~@fepegar I guess torchio does not support the symlink for path XD~~

EDIT: it was due to a permission issue

**Comment by fepegar** [comment]
(For reference: it does. It was just a permissions issue)

**Comment by fepegar** [comment]
Please @mathpluscode read carefully the following before submitting anything:
- [OASIS Data Use Agreement](https://www.oasis-brains.org/#access)
- [ADNI "Access data and samples"](http://adni.loni.usc.edu/data-samples/access-data/)
